src/combobox/ComboBox.ts

```diff
@@ -304,10 +305,20 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 		} = this.properties;
 
 		if (
-			!this.registry ||
+			!this._registry ||
 			includes(evt.changedPropertyKeys, 'customResultItem') ||
 			includes(evt.changedPropertyKeys, 'customResultMenu')) {
-			this.registry = this._createRegistry(customResultItem, customResultMenu);
+
+			if ( !this._registry || includes(evt.changedPropertyKeys, 'customOption')) {
```

Comment:
What is `customOption`? Do we need to check for `this._registry` twice? A little confused by these lines. Maybe they're left over from Select refactoring?